draft-packages/divider/KaizenDraft/Divider/styles.module.scss

```diff
@@ -1,27 +1,50 @@
-@import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color-vars";
+@import "~@kaizen/design-tokens/sass/border-vars";
+@import "~@kaizen/design-tokens/sass/variable-identifiers";
+
+$dt-content-border-color: rgba(
+  var($kz-var-id-color-slate-rgb-params, $kz-var-color-wisteria-700-rgb-params),
+  0.1
+);
 
 .wrapper {
   width: 100%;
   border: 0;
   margin: 0;
-  border-radius: $kz-border-solid-border-radius;
+  border-radius: $kz-var-border-solid-border-radius;
   // This is here to protect against a global style in a consumer.
   // https://github.com/cultureamp/murmur/blob/master/app/assets/stylesheets/legacy/modules/_body.scss
   visibility: visible;
 }
 
 .content {
-  border-top: 1px solid rgba($kz-color-wisteria-700, 0.1);
+  border-top: 1px solid;
+  border-color: $dt-content-border-color;
+
   &.reversed {
-    border-color: rgba($kz-color-white, 0.1);
+    border-color: rgba($kz-var-color-white-rgb-params, 0.1);
   }
 }
 
 .canvas {
-  border-top: 2px solid rgba($kz-color-wisteria-700, 0.2);
-  border-bottom: 1px solid rgba($kz-color-wisteria-700, 0.2);
+  --dt-canvas-border-color-zen: rgba(
+    #{$kz-var-color-wisteria-700-rgb-params},
+    0.2
+  );
+  --dt-canvas-border-color-heart: rgba(
+    var(#{$kz-var-id-color-slate-rgb-params}),
+    0.1
+  );
+  --dt-canvas-border-color: var(
+    --dt-canvas-border-color-heart,
+    var(--dt-canvas-border-color-zen)
+  );
```

Comment:
I didn't realise you can define the custom property on the same element you consume it on. Nice!